packages/interaction-kit/src/definitions/application-commands.ts

```diff
@@ -30,17 +34,21 @@ export enum ApplicationCommandType {
 	MESSAGE = 3,
 }
 
-/** @link https://discord.com/developers/docs/interactions/slash-commands#application-command-object-application-command-option-structure */
+/** @link https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-option-structure */
 export type ApplicationCommandOption = {
 	type: ApplicationCommandOptionType;
 	name: string;
 	description: string;
 	required?: boolean;
 	choices?: ApplicationCommandOptionChoice[];
 	options?: ApplicationCommandOption[];
+	channel_types?: ChannelType[],
+	min_value?: number,
+	max_value?: number,
+	autocomplete?: boolean
```

Comment:
There should be semicolons instead of commas to make it consistent.
```suggestion
	channel_types?: ChannelType[];
	min_value?: number;
	max_value?: number;
	autocomplete?: boolean;
```